app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -279,10 +291,56 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             createIVButton();
             createArcPointer();
             createArcAdjuster();
+            /* Assumes MainActivity initialized ScreenGrabber before starting this service. */
```

Comment:
I'm fine with this behavior and the assumption, I'd just prefer the source to read different with `getInstance()` to show it more clearly. But not so important.